src/ui/redux/reducers/subscriptions.js

```diff
@@ -134,14 +134,6 @@ export default handleActions(
       ...state,
       loadingSuggested: false,
     }),
-    [ACTIONS.SUBSCRIPTION_FIRST_RUN_COMPLETED]: (state: SubscriptionState): SubscriptionState => ({
-      ...state,
-      firstRunCompleted: true,
-    }),
-    [ACTIONS.VIEW_SUGGESTED_SUBSCRIPTIONS]: (state: SubscriptionState): SubscriptionState => ({
```

Comment:
Both of these can be moved into their respective components and use persisted local state to manage if we should show them or not.